src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```
@@ -5,6 +5,14 @@
         <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);DOTNET_TFM-windows10.0.19041.0</TargetFrameworks>
         <!-- Uncomment to also build the tizen app. You will need to install tizen by following this: https://github.com/Samsung/Tizen.NET -->
         <!-- <TargetFrameworks>$(TargetFrameworks);DOTNET_TFM-tizen</TargetFrameworks> -->
+
+         <!-- Note for MacCatalyst:
+            The default runtime is maccatalyst-x64, except in Release config, in which case the default is maccatalyst-x64;maccatalyst-arm64.
+            When specifying both architectures, use the plural <RuntimeIdentifiers> instead of the singular <RuntimeIdentifer>.
+            The App Store will NOT accept apps with ONLY maccatalyst-arm64 indicated;
```

```suggestion
            The Mac App Store will NOT accept apps with ONLY maccatalyst-arm64 indicated;
```

In case any reader is unfamiliar with the generic sounding term "App Store," I suggest adding "Mac" to the name.

The Apple App Store guidelines specifically mention that either form is acceptable, but I think the more specific one could be more helpful.

Guidelines: https://developer.apple.com/app-store/marketing/guidelines/

> App Store
> Always typeset App Store with an uppercase A and an uppercase S followed by lowercase letters.
> 
> Refer to just the App Store unless you need to be more specific; in that case, you can use App Store for iPhone, App Store for iPad, Mac App Store, App Store for Apple TV, App Store for Apple Watch, or App Store for iMessage. To refer to all the versions, use this order: App Store for iPhone, iPad, Mac, Apple Watch, Apple TV, and iMessage. Don’t use terms such as Apple Watch App Store or App Store for watchOS.